Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers_test.py

```diff
@@ -41,10 +41,83 @@ def test_risky_users_list_command(requests_mock) -> None:
     mock_response = load_mock_response('list_risky_users.json')
     requests_mock.post(ACCESS_TOKEN_REQUEST_URL, json={})
     requests_mock.get(f'{BASE_URL}identityProtection/riskyUsers', json=mock_response)
-    result = risky_users_list_command(mock_client(), {'limit': '20', 'page': '1'})
-    assert result.outputs_prefix == 'AzureRiskyUsers.RiskyUser'
-    assert result.outputs_key_field == 'id'
-    assert len(result.raw_response) == 3
+    result = risky_users_list_command(mock_client(), {'limit': '20'})
+    assert result[0].outputs_prefix == 'AzureRiskyUsers.RiskyUser'
+    assert result[0].outputs_key_field == 'id'
+    assert len(result[0].raw_response) == 3
+
+
+def test_risky_users_list_command_with_page_size(requests_mock) -> None:
+    """
+    Scenario: List Risky Users.
+    Given:
+     - User has provided valid credentials.
+     - Headers and JWT token have been set.
+    When:
+     - risky_users_list_command is called.
+    Then:
+     - Ensure number of items is correct.
+     - Ensure outputs prefix is correct.
+     - Ensure outputs key fields is correct.
+    """
+    from AzureRiskyUsers import risky_users_list_command
+    mock_response = load_mock_response('list_risky_users.json')
+    requests_mock.post(ACCESS_TOKEN_REQUEST_URL, json={})
+    requests_mock.get(f'{BASE_URL}identityProtection/riskyUsers', json=mock_response)
+    result = risky_users_list_command(mock_client(), {'page_size': '3'})
+    assert result[0].outputs_prefix == 'AzureRiskyUsers.RiskyUser'
+    assert result[0].outputs_key_field == 'id'
+    assert len(result[0].raw_response) == 3
+    assert result[1].outputs == {'AzureRiskyUsers(true)': {'RiskyUserListNextToken': mock_response.get('@odata.nextLink')}}
+
+
+def test_risky_users_list_command_with_token(requests_mock) -> None:
+    """
+    Scenario: List Risky Users.
+    Given:
+     - User has provided valid credentials.
+     - Headers and JWT token have been set.
+    When:
+     - risky_users_list_command is called.
+    Then:
+     - Ensure number of items is correct.
+     - Ensure outputs prefix is correct.
+     - Ensure outputs key fields is correct.
+    """
+    from AzureRiskyUsers import risky_users_list_command
+    mock_response = load_mock_response('list_risky_users.json')
+    requests_mock.post(ACCESS_TOKEN_REQUEST_URL, json={})
+    requests_mock.get(f"{BASE_URL}/token", json=mock_response)
+    result = risky_users_list_command(mock_client(), {'page_size': '3', "next_token": f"{BASE_URL}/token"})
+    assert result[0].outputs_prefix == 'AzureRiskyUsers.RiskyUser'
+    assert result[0].outputs_key_field == 'id'
+    assert len(result[0].outputs) == 3
+    assert len(result[0].raw_response) == 3
+    assert result[1].outputs == {'AzureRiskyUsers(true)': {'RiskyUserListNextToken': mock_response.get('@odata.nextLink')}}
```

Comment:
Lets add an assertion to validate that the client method was called with the next_token and not with the page_size.